configs/README.md

```diff
@@ -27,6 +27,7 @@ Refer to the tables below for descriptions of all fields in the configuration fi
 |data-order| array[string] | **REQUIRED**  input data order. Data order: *C* (row-major, default) or *F* (column-major) |
 |dtype| array[string] | **REQUIRED**  input data type. Data type: *float64* (default) or *float32* |
 |check-finitness| array[] | Check finiteness in sklearn input check(disabled by default) |
+|device| array[string] | For scikit-learn only. The list of devices to run the benchmarks on. It can be *None* (default, run on CPU without sycl context) or one of the types of sycl devices: *cpu*, *gpu*, *host*. Please refer to [SYCL specification](https://www.khronos.org/files/sycl/sycl-2020-reference-guide.pdf) for details|
```

Comment:
```suggestion
|device| array[string] | For scikit-learn only. The list of devices to run the benchmarks on. It can be *None* (default, run on CPU without sycl context) or one of the types of sycl devices: *cpu*, *gpu*, *host*. Refer to [SYCL specification](https://www.khronos.org/files/sycl/sycl-2020-reference-guide.pdf) for details.|
```